Service/RateLimitInfoManager.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Noxlogic\RateLimitBundle\Service;
+
+use Noxlogic\RateLimitBundle\Annotation\RateLimit;
+
+class RateLimitInfoManager
```

Comment:
why do we need this separate from `RateLimitService`? maybe it should just be part of that.

Reply:
I also agree with this. Having `RateLimitInfoManager` and `RateLimitService` is concessionary

Reply:
Indeed, good point, thanks